spec/interactors/article/combine_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+
+describe Article::Combine do
+  let(:user) { User.find_by(login: OWNER) }
+  let!(:collection) { create(:collection, owner_user_id: user.id) }
```

Comment:
What's the difference between `let` and `let!`?

Reply:
It eager saves, and does validations too. sometimes when no ! the object is not created when specs is run, or that it tried to  save, no error thrown but object failed to save so specs will still run with said objects that are not created.